packages/contracts/contracts/plugin/PluginSetupProcessor.sol

```diff
@@ -517,14 +509,7 @@ contract PluginSetupProcessor is DaoAuthorizable {
 
         validatePreparedSetupId(pluginInstallationId, preparedSetupId);
 
-        // Store only the data required for subsequent updates or uninstallations and set the rest to zero.
-        bytes32 newSetupId = _getSetupId(
-            _params.pluginSetupRef,
-            ZERO_BYTES_HASH,
-            _params.helpersHash,
-            bytes(""),
-            PreparationType.None
-        );
+        bytes32 appliedSetupId = _getAppliedSetupId(_params.pluginSetupRef, _params.helpersHash);
```

Comment:
Is this change part of the rebase/merge? This is changing the behavior

Reply:
I cannot see this change on the current commit. Maybe GitHub view was messed up? Can You recheck?